### ISSUE TITLE
Issue with the verification of cleartext signed message

### ISSUE THREAD

**Issue by BRULE-Florian** [vanitasvitae/pgpainless#219]
### Discussed in https://github.com/pgpainless/pgpainless/discussions/217

<div type='discussions-op-text'>

<sup>Originally posted by **BRULE-Florian** November 23, 2021</sup>
When I call `PGPainless.verifyCleartextSignedMessage()` with an `Inputstream` that contains a **too long** _cleartext_ signed message (**maximum 8191 characters allowed**), it doesn't work correctly.
It throws `java.io.IOException: Resetting to invalid mark`.

Here is the code that causes the exception:
```
String signedMessage = "...";
BufferedInputStream signedMessageStream = new BufferedInputStream(
    new ByteArrayInputStream(signedMessage.getBytes(StandardCharsets.UTF_8)));

ConsumerOptions options = new ConsumerOptions()
    .addVerificationCert(publicKey)
    .addVerificationOfDetachedSignature(signature);

DecryptionStream verificationStream = PGPainless.verifyCleartextSignedMessage()
    .onInputStream(signedMessageStream)
    .withStrategy(new InMemoryMultiPassStrategy())
    .withOptions(options)
    .getVerificationStream();
```
_The exception is thrown in the method `getVerificationStream()`._
</div>

**Comment by BRULE-Florian** [comment]
pgpainless version: 0.2.19

**Comment by BRULE-Florian** [comment]
Related to the issue #220

**Comment by vanitasvitae** [comment]
I just released PGPainless 1.0.0-rc5 which should fix this issue.
